composer.json

```diff
@@ -18,13 +18,13 @@
     "ergebnis/license": "^2.6.0",
     "ergebnis/php-cs-fixer-config": "^6.46.0",
     "phpspec/prophecy": "^1.7.0",
-    "phpspec/prophecy-phpunit": "^2.3",
+    "phpspec/prophecy-phpunit": "^2.3 || dev-master#d3c2804 as 2.4.0",
```

Comment:
A simpler workaround would have been `^2.3@dev`, as phpstan-prophecy has the proper branch alias for its master branch (and the commit hash lock is a very bad idea due to how this actually works in composer)